src/api/spec/models/comment_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe Comment do
+  let!(:comment) { create(:comment) }
```

Comment:
You don't need the `!`

In case that you need the object to be created (like happens in the `blank_or_destroy` test) you can do

``` Ruby
  describe "blank_or_destroy" do
    before do
      comment
    end
```
That way you don't create it for all tests.